Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -2472,8 +2490,29 @@ void MainWindow::processMyselfJoined(int channelid)
 
 void MainWindow::processMyselfLeft(int channelid)
 {
-    Q_UNUSED(channelid);
-    m_mychannel = {};
+    TT_GetChannel(ttInst, channelid, &m_mychannel);

```

Comment:
This call may fail if user is last in channel and channel is removed.

Also "m_mychannel" has to be reset otherwise the client will appear to still be in a channel (m_mychannel will appear valid).